docs/source/whatsnew/pr/remove-pydb.rst

```diff
@@ -0,0 +1 @@
+Integration with pydb has been removed since pydb development has been stopped since 2012, and pydb is not installable from PyPI
```

Comment:
I think this should be an `incompat-` file, so it winds up in the incompatible changes section of the release notes.